include/seqan3/alignment/pairwise/policy/affine_gap_banded_policy.hpp

```diff
@@ -72,24 +72,53 @@ class affine_gap_banded_policy : public affine_gap_policy<affine_gap_banded_poli
     {
         using std::get;
         // Unpack the cached variables.
-        auto & [current_entry, next_entry] = current_cell;
-        auto & [main_score, hz_score] = current_entry;
-        auto const & prev_hz_score = get<1>(next_entry);
+        auto & [score_entry, coordinate, trace_value] = current_cell;
+        auto & [current_entry, next_entry] = score_entry;
+        auto & [main_score, hz_score, hz_trace] = current_entry;
+        auto const & [prev_main_score, prev_hz_score, prev_hz_trace]= next_entry;
         auto & [prev_cell, gap_open, gap_extend, opt] = cache;
-        auto & [tmp, vt_score] = prev_cell;
+        auto & [tmp, vt_score, vt_trace] = prev_cell;
+
+        std::ignore = prev_main_score;
```

Comment:
The variable is not used and would otherwise create an unused warning. This is now a just more expressive way to say `(void) prev_main_score`, but with the same effect and optimised away by the compiler anyway.

Reply:
`(void) prev_main_score` is guaranteed to not generate code, while the assignment is only likely to not generate code. I would prefer the former...